src/integration/pages/radioBrandPage/commonBrandTests.js

```diff
@@ -0,0 +1,37 @@
+export default () => {
+  it('I can see the brand title', () => {
```

Comment:
Is it worth trying to try and reuse the commonEpisode tests from the onDemandRadioPage integration tests? There's a lot of duplication here

Reply:
The original idea was to add the brand tests to the current radio tests. Jonathan wanted them separate so as not to complicate another PR.

Reply:
apologies I meant to keep the pages separate. there is a common folder for common tests